core/modules/system/tests/system.test

```diff
@@ -561,42 +561,42 @@ class ModuleVersionTestCase extends ModuleTestCase {
    */
   function testModuleVersions() {
     $dependencies = array(
-      // All comparisons are done against 1.x-2.4-beta3.
+      // All comparisons are done against 1.x-2.4.5-beta3.
       'common_test' => TRUE,
       // Branch incompatibility.
-      'common_test (1.x)' => FALSE,
+      'common_test (1.x.x)' => FALSE,
       // Branch compatibility.
-      'common_test (2.x)' => TRUE,
+      'common_test (2.x.x)' => TRUE,
       // Another branch incompatibility.
-      'common_test (>2.x)' => FALSE,
+      'common_test (>2.x.x)' => FALSE,
       // Another branch compatibility.
-      'common_test (<=2.x)' => TRUE,
+      'common_test (<=2.x.x)' => TRUE,
       // Another branch incompatibility.
-      'common_test (<2.x)' => FALSE,
+      'common_test (<2.x.x)' => FALSE,
       // Another branch compatibility.
-      'common_test (>=2.x)' => TRUE,
+      'common_test (>=2.x.x)' => TRUE,
```

Comment:
I think all these checks above should probably be reverted. A module specifying a dependency would only declare `>=2.x`. And the subsequent patch `.x` would be implied. You'd never see a `>=2.x.1` for example.

Reply:
That makes sense. I'll change those to your suggestion